docs/product/components/usercard.html

```diff
@@ -0,0 +1,189 @@
+            <li class="s-usercard--rep">
+                2,500k
+            </li>
+    
+            <li class="s-award-bling s-award-bling__gold">
+                5
+            </li>
+    
+            <li class="s-award-bling s-award-bling__silver">
+                9
+            </li>
+    
+            <li class="s-award-bling s-award-bling__bronze">
+                1
+            </li>
+        </ul>
+    </div>
+
+{% endhighlight %}
+        <div class="stacks-preview--example overflow-x-auto">
+            <div class="s-usercard">
+                <time class="s-usercard--time">3 mins ago</time>
+            
+                <a href="#" class="s-usercard--avatar s-avatar s-avatar__md">
+                    <img src="https://placekitten.com/32/32" />
+                </a>
+            
+                <a href="#">
+                    <h4 class="s-usercard--name">Paul Wright</h4>
+                </a>
+            
+                <ul class="list-reset s-usercard--awards">
+                    <li class="s-usercard--rep">
+                        2,500k
+                    </li>
+            
+                    <li class="s-award-bling s-award-bling__gold">
+                        5
+                    </li>
+            
+                    <li class="s-award-bling s-award-bling__silver">
+                        9
+                    </li>
+            
+                    <li class="s-award-bling s-award-bling__bronze">
+                        1
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="stacks-section">
+        {% header "h2", "Highlighted" %}
+    <div class="stacks-preview">
+{% highlight html %}
+    <div class="s-usercard s-usercard--highlighted">
+        <time class="s-usercard--time">3 mins ago</time>
+    
+        <a href="#" class="s-usercard--avatar s-avatar s-avatar__md">
+            <img src="https://placekitten.com/32/32" />
+        </a>
+    
+        <a href="#">
+            <h4 class="s-usercard--name">Paul Wright</h4>
+        </a>
+    
+        <ul class="list-reset s-usercard--awards">
+            <li class="s-usercard--rep">
+                2,500k
+            </li>
+    
+            <li class="s-award-bling s-award-bling__gold">
+                5
+            </li>
+    
+            <li class="s-award-bling s-award-bling__silver">
+                9
+            </li>
+    
+            <li class="s-award-bling s-award-bling__bronze">
+                1
+            </li>
+        </ul>
+    </div>
+{% endhighlight %}
+        <div class="stacks-preview--example overflow-x-auto">
+            <div class="s-usercard s-usercard--highlighted wmx2">
+                <time class="s-usercard--time">3 mins ago</time>
+            
+                <a href="#" class="s-usercard--avatar s-avatar s-avatar__md">
+                    <img src="https://placekitten.com/32/32" />
```

Comment:
We'll want to round these with `br-sm`. Do we need to add child img rounding to `.avatar`?